### PR TITLE
WIP: add tests for toolbar/toolbar.html example

### PR DESCRIPTION
**Pull request by spectranaut**
Waiting on questions on toolbar review: https://github.com/w3c/aria-practices/issues/539#issuecomment-413953446

Also blocking on acceptance of race condition PR: #837

### CODE REVIEW

**Comment by mcking65** [comment]
Raised issue #847 for the toolbar bugs. Of the 6 bugs listed in #847, how many are caught by these tests?

**Comment by spectranaut** [comment]
@mcking65, currently 2 -- I only wrote two tests (left arrow and right arrow) for the issues I had questions with. I can write the remaining tests and comment them out as we discussed.

**Comment by spectranaut** [comment]
The last two commits add tests (expect for the "aria-disabled") and comment them out, with a reference to #847

**Comment by mcking65** [comment]
@jessebeach, @tatermelon, @sh0ji, even though travis is failing, this is ready for review. The failures are related to a race condition that is not specific to these tests. @spectranaut is debugging the failures separately.

File: test/tests/toolbar.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+      return tabindex === '0';
+    },
+    500,
+    'click did not set tabindex="0" on: ' + selector
+  ).catch((err) => { return err; });
+};
+
+const waitAndCheckFocus = async function (t, selector) {
+  return t.context.session.wait(
+    async function () {
```
Comment (jessebeach) [inline test/tests/toolbar.js]: Does this need to be async? Everything in the function body is synchronous, unless `executeScript` is missing an `await`?

**Pull request merged**